src/sidebar/store/modules/session.js

```diff
@@ -1,12 +1,18 @@
 'use strict';
 
+const serviceConfig = require('../../service-config');
 const util = require('../util');
 
-function init() {
+function init(settings) {
   /**
    * Profile/session information for the active user.
    */
+  const service = serviceConfig(settings) || {};
   return {
+    /** Whether handlers provided in service configuration for certain events */
+    configuredServiceEventHandlers: {
+      onHelpRequest: service.onHelpRequestProvided,
```

Comment:
Since the value of `onHelpRequestProvided` never changes within the lifetime of the application, it is arguable whether there is value in putting it in the store or not.

Generally I would say that state only needs to live in the store if:

 - It can change during the lifetime of the application, resulting in a need to re-render components and trigger other side-effects OR
 - It is needed in order for reducers to produce an updated state, since reducers are supposed to be pure

If within a component you need to combine some static state with state from the store you can do that in a helper that takes the store state + static state as input.

The advantage of keeping static state outside of the store is that it then has fewer dependencies (no need to create a store instance) which means you can more easily use or adapt it for use it in different contexts, writing tests is easier.